etc/telegraf.conf

```diff
@@ -430,8 +430,42 @@
 #   # listen = ":9126"
 
 
-# # Configuration for the Riemann server to send metrics to
+# # Configuration for Riemann server to send metrics to
 # [[outputs.riemann]]
+#   ## Address of the Riemann server
+#   address = "localhost:5555"
+#
+#   ## Transport protocol to use, either tcp or udp
+#   transport = "tcp"
```

Comment:
I could, but then I'd have to split/parse it to take it apart again anyway, since the Riemann library needs the protocol and address as separate parameters. `address` is not used anywhere else otherwise. It's not a complicated thing to do, but I don't see the benefit of it?